#!/usr/bin/env python
PACKAGE = "turtlebot_trajectory_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#gen.add("int_param",    int_t,    0, "An Integer parameter", 50,  0, 100)
gen.add("k_turn", double_t, 0, "Gain on angular error",    1, 0,   100)
gen.add("k_drive_x", double_t, 0, "Gain on x error",    1, 0,   100)
gen.add("k_drive_y", double_t, 0, "Gain on y error",    1, 0,   100)


exit(gen.generate(PACKAGE, "turtlebot_controller", "TurtlebotController"))
